### ISSUE TITLE
Preserve closing newline?

### ISSUE THREAD

**Issue by smlombardi** [badsyntax/SassBeautify#78]
Running the beautifier clips the closing newline from the file, which in turn is flagged as a warning by scss-lint.

Any way to get an option to preserve/add closing newline?

**Comment by akloeber** [comment]
+1

**Comment by badsyntax** [comment]
Have you tried setting `"ensure_newline_at_eof_on_save": true` in your user preferences?

**Comment by akloeber** [comment]
You mean in `SassBeautify.sublime-settings` because the other options are camel-cased? That still does not work.

**Comment by badsyntax** [comment]
No I mean in the sublime user settings / preferences.

**Comment by akloeber** [comment]
Greats that works. I wasn't aware of a global config setting for that.

**Comment by lombardi-gux** [comment]
Yes, the preference adds it back in at save. But this sequence happens every day:

1 - run beautify, it removes the newline
2 - sass lint flags the error
3 - save file, adds newline again

It works, but it's annoying.

**Comment by badsyntax** [comment]
The beautify plugin should be saving the file after it's beautified, so the linter shouldn't' be complaining, but I'd need to test this to see exactly whats happening. When i get some time I'll have a go with the scss-lint plugin.

**Comment by davetgreen** [comment]
Having the newline preserved would be superb. +1

**Comment by mfcovington** [comment]
@badsyntax Thanks for the `ensure_newline_at_eof_on_save` tip!

**Comment by gregmatys** [comment]
+1
